packages/cli/src/lib/snapshot/reportViewer.ts

```diff
@@ -42,43 +66,62 @@ export class ReportViewer {
   }
 
   // TODO: Change logic once SRC-4448 is complete
-  private printTableSection(row: {
-    resourceName: string;
-    operations: ResourceSnapshotsReportOperationModel;
-  }) {
+  private printTableSection(
+    row: {
+      resourceName: string;
+      operations: ResourceSnapshotsReportOperationModel;
+    },
+    indentation = 3
+  ) {
     const resourceType = this.prettyPrintResourceName(row.resourceName);
-    let output = `   ${resourceType}\n`;
+    let output = `${''.padStart(indentation)}${resourceType}\n`;
 
-    if (row.operations.resourcesCreated > 0) {
+    if (
+      this.operationsToDisplay.resourcesCreated &&
```

Comment:
Suggestion: For readability sake, since it's all the same check but just on a different section of the report we could add a :

```
private shouldDisplayOperation(operationsToDisplaySection, operationSection): boolean {
  [...]
}

if(shouldDisplayOperation(this.operationsToDisplay.resourcesCreated, row.operations.resourcesCreated) {
   [...]
}
```